src/main/java/com/github/rholder/retry/Clock.java

```diff
@@ -0,0 +1,5 @@
+package com.github.rholder.retry;
+
+public interface Clock {
```

Comment:
Maybe use the Guava [`Ticker`](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/Ticker.html) interface instead?